src/components/Explore/utils.ts

```diff
@@ -4,6 +4,8 @@ import {
   range as _range,
   deburr as _deburr,
   words as _words,
+  isNumber,
+  dropRightWhile,
```

Comment:
I do it sometimes to avoid confusion between globals such as `isNan` from lodash versions, but we probably don't need the `_` prefix in most cases, removing!